### DISCUSSION TITLE
✅ 🔗 🚀 Feature Request: Allow to set Additional Breakpoints in the nested Tabs widget (Related to #21202) - [ED-9306]

### DISCUSSION THREAD

**Discussion by lpsct** [elementor/elementor#22861] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Tab widget only turns accordion on mobile view

### Describe the solution you'd like

It would be great to be able to choose between tab or accordion view on responsive additional breakpoints. Right now a tab widget only turns accordion on mobile view. but I have 2 other additional breakpoints that don't look good on tabs view.

**Comment by vladninja** [comment]
Duplcate: https://github.com/orgs/elementor/discussions/21202

Same problem for me.
Case:
Inside the tab, place the Loop Grid widget with infinite scroll -> The user not be able to show other accordions (need to scroll when no more posts will be loaded). So the solution here is prevent to make accordions instead of Tabs.

As a workaround, can solve that by adding custom CSS, which prevent changes on mobile:
```
.e-n-tabs-mobile>.elementor-widget-container>.e-n-tabs>.e-n-tabs-heading {
	display: var(--n-tabs-heading-display) !important;
}

.e-n-tabs-mobile>.elementor-widget-container>.e-n-tabs>.e-n-tabs-content>.e-collapse {
	display: none !important;
}
```

**Comment by loucasl** [comment]
Wasn't this feature just added in 3.14?

**Comment by nicholaszein** [comment]
Hello there @lpsct! 

### This feature was released with Elementor v3.14! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
